src/gui/parameters_screen.py

```diff
@@ -29,48 +27,75 @@ def __init__(self, parent):
             self.cells_side_count_latitude = 10
             self.cells_side_count_longitude = 10
             self.point_side_size = 50
-            self.correctly_set_parameters = [1, 1, 1, 1, 1, 1, 1, 1, 1, 1]
+            self.iter_as_sec = 20
```

Comment:
These values initial state should be set in constants and then imported from there.  It's good to have it in a place where it's easy to change, not in the middle of the gui